test/kt/CMakeLists.txt

```diff
@@ -124,7 +124,49 @@ if (ONEDPL_TEST_ENABLE_KT_ESIMD)
     _generate_esimd_sort_test("256" "32" "double" "" 1000) # segfault
 endif()
 
+function (_generate_scan_test _data_per_work_item _work_group_size _type)
+
+    if ((NOT TARGET build-scan-tests) AND (NOT TARGET run-scan-tests))
+        add_custom_target(build-scan-tests COMMENT "Build all scan kernel template tests")
+        add_custom_target(run-scan-tests
+            COMMAND "${CMAKE_CTEST_COMMAND}" -R "^run-scan-tests$" --output-on-failure --no-label-summary
+            DEPENDS build-scan-tests
+            COMMENT "Build and run all scan kernel template tests")
+    endif()
+
+    string(REPLACE "_t" "" _type_short ${_type})
+    set(_target_name "single_pass_scan_dpwi${_data_per_work_item}_wgs${_work_group_size}_${_type_short}")
```

Comment:
single_pass_scan -> igpu_scan

Reply:
It's named now according to cpp filename